cloud/google/gceproviderconfig/types.go

```diff
@@ -30,4 +30,15 @@ type GCEProviderConfig struct {
 
 	// The name of the OS to be installed on the machine.
 	OS string `json:"os"`
+	Image       string `json:"image"`
```

Comment:
The OS is what eventually maps to the OS Image. Why is this field being introduced in this PR?

Reply:
This was added as an error with resolving a merge conflict. Removing.